src/pages/ReportDetail/Page.tsx

```
@@ -1,263 +1,233 @@
+   }
+
+   // TODO: 개선 필요
+   // state 로 처리하는 걸 없애고 싶다..
+   // 관리자는 접근 가능.
+   // 회원은 자기 페이지만 접근 가능.
+   // eslint-disable-next-line react-hooks/rules-of-hooks
+   const { data: report, isLoading } = useQuery(
+      ['report', id],
+      () => {
+         if (isAdmin) {
+            return readReportDetail(+id);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)